client/lib/site/index.js

```diff
@@ -86,46 +87,7 @@ Site.prototype.set = function( attributes ) {
  * Update any computed attributes
  */
 Site.prototype.updateComputedAttributes = function() {
-	/**
```

Comment:
I'm leaving `config` in, as it uses `is_previewable`, we'll move it out and use https://github.com/Automattic/wp-calypso/pull/7248 once we've `connect`ed the sidebar.